src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
sidebar: i think our current list of browsers that ideally should be supported is...

Firefox/Nightly (desktop; do you plan to support mobile?)
Chrome/Chromium (desktop and at least Daydream mobile)
Oculus Browser (Carmel is deprecated)
Samsung Internet

agree?  so Firefox behavior alone may not suffice, both Chromium and Firefox is probably good enough in most cases since at present Oculus and Samsung browsers generally follow...